params/params_mixnet.go

```diff
@@ -56,13 +56,12 @@ var MixNetParams = Params{
 		CuckatooMinDifficulty:  0x1600000, // 96
 		CuckaroomMinDifficulty: 0x1600000, // 96
 
-		Percent: []pow.Percent{
-			{
-				Blake2bDPercent:         20,
-				CuckarooPercent:         30,
-				QitmeerKeccak256Percent: 30,
-				CuckaroomPercent:        20,
-				MainHeight:              0,
+		Percent: map[pow.MainHeight]map[pow.PowType]pow.PercentValue{
```

Comment:
avoid the cumbersome type declaration like `map[pow.MainHeight]map[pow.PowType]pow.PercentValue`
covert it into more understandable type declaration like 
```go
map[pow.MainHeight]pow.Percent{
   ...
}
```
inside the `pow` package
```go
type MainHeight uint32
type Percent map[pow.PowType]pow.PercentValue
```
BTW, alway try to use the exact integer type and fixed storage size as you can. since the MainHeight can't be negative, you should not declare as an signed type, and do the additional error test like
```go
if mHeight < 0 {
	return errors.New("pow config error, must greater than or equal to 0!")				
}
```